plugins/status/plugin_test.go

```diff
@@ -100,7 +105,8 @@ func TestPluginStartBulkUpdate(t *testing.T) {
 	status := testStatus()
 
 	fixture.plugin.BulkUpdateBundleStatus(map[string]*bundle.Status{status.Name: status})
-	result = <-fixture.server.ch
+	<-fixture.server.ch // Discard first request.
```

Comment:
❓ Should this not go in line 87? Where `result := <-fixture.server.ch` was removed?